### DISCUSSION TITLE
PG_cron not executing function

### DISCUSSION THREAD

**Discussion by Maxson52** [supabase/supabase#2066] [Questions] [answered]
I've been using Supabase with a Svelte project of mine for some time now and have really been loving it! Just recently, I started tinkering with Postgres as I have never used it before. This project keeps track of your total XP (earned from various goals on the site) and your weekly XP. Here comes the issue. I think I should be using pg_cron to reset the weekly xp every week, but the following doesn't seem to be doing anything.
```
create extension if not exists pg_cron;

create or replace function public.handle_xp_reset() 
returns bigint as $$
begin
UPDATE public.stats SET weekly_xp=0;

return 0;
end;
$$ language plpgsql security definer;


SELECT cron.schedule('xp_reset', '* * * * *', 'SELECT handle_xp_reset()');
```
For testing purposes, I'm using a minutely schedule.
The function seems to work when called by itself, but the cron job doesn't seem to do it.
Also, everything is being done from the SQL editor in the Supabase web app.

Bear with me, I've never used a query language before nor did I use any sort of tutorial for this.

**Comment by inian** [comment, marked as answer]
Hi @Maxson52, we are tracking this issue here https://github.com/supabase/supabase/issues/1703
